feature/gribi/otg_tests/base_leader_election_test/base_leader_election_test.go

```diff
@@ -128,55 +144,88 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 }
 
 // configureATE configures port1, port2 and port3 on the ATE.
-func configureATE(t *testing.T, ate *ondatra.ATEDevice) *ondatra.ATETopology {
-	top := ate.Topology().New()
-
-	p1 := ate.Port(t, "port1")
-	i1 := top.AddInterface(atePort1.Name).WithPort(p1)
-	i1.IPv4().
-		WithAddress(atePort1.IPv4CIDR()).
-		WithDefaultGateway(dutPort1.IPv4)
-
-	p2 := ate.Port(t, "port2")
-	i2 := top.AddInterface(atePort2.Name).WithPort(p2)
-	i2.IPv4().
-		WithAddress(atePort2.IPv4CIDR()).
-		WithDefaultGateway(dutPort2.IPv4)
-
-	p3 := ate.Port(t, "port3")
-	i3 := top.AddInterface(atePort3.Name).WithPort(p3)
-	i3.IPv4().
-		WithAddress(atePort3.IPv4CIDR()).
-		WithDefaultGateway(dutPort3.IPv4)
-
+func configureATE(t *testing.T, ate *ondatra.ATEDevice) gosnappi.Config {
+
+	otg := ate.OTG()
+	top := otg.NewConfig(t)
+
+	top.Ports().Add().SetName(atePort1.Name)
+	dev := top.Devices().Add().SetName(atePort1.Name)
+	eth := dev.Ethernets().Add().SetName(atePort1.Name + ".Eth")
+	eth.SetPortName(dev.Name()).SetMac(atePort1.MAC)
+	ip := eth.Ipv4Addresses().Add().SetName(dev.Name() + ".IPv4")
+	ip.SetAddress(atePort1.IPv4).SetGateway(dutPort1.IPv4).SetPrefix(int32(atePort1.IPv4Len))
+
+	top.Ports().Add().SetName(atePort2.Name)
+	dev = top.Devices().Add().SetName(atePort2.Name)
+	eth = dev.Ethernets().Add().SetName(atePort2.Name + ".Eth")
+	eth.SetPortName(dev.Name()).SetMac(atePort2.MAC)
+	ip = eth.Ipv4Addresses().Add().SetName(dev.Name() + ".IPv4")
+	ip.SetAddress(atePort2.IPv4).SetGateway(dutPort2.IPv4).SetPrefix(int32(atePort2.IPv4Len))
+
+	top.Ports().Add().SetName(atePort3.Name)
+	dev = top.Devices().Add().SetName(atePort3.Name)
+	eth = dev.Ethernets().Add().SetName(atePort3.Name + ".Eth")
+	eth.SetPortName(dev.Name()).SetMac(atePort3.MAC)
+	ip = eth.Ipv4Addresses().Add().SetName(dev.Name() + ".IPv4")
+	ip.SetAddress(atePort3.IPv4).SetGateway(dutPort3.IPv4).SetPrefix(int32(atePort3.IPv4Len))
+
+	otg.PushConfig(t, top)
+	otg.StartProtocols(t)
 	return top
 }
 
+func waitOtgArpEntry(t *testing.T) {
```

Comment:
waitOtgArpEntry -> waitOTGARPEntry because of golang initialism.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms